src/bundle.ts

```diff
@@ -32,14 +33,23 @@ export class BuildBundler extends Transform {
   bundler: Bundler;
   sharedFile: File;
 
+  files = new Map<string, File>();
+
   constructor(config: ProjectConfig, analyzer: BuildAnalyzer) {
-    super({objectMode: true});
+    super({objectMode: true, highWaterMark: 10000});
```

Comment:
Removed this setting.  I misunderstood its example in the BuildAnalyzer Transform.